journal-of-international-relations-and-development.csl

```diff
@@ -0,0 +1,223 @@
+  <info>
+    <title>Journal of International Relations and Development</title>
+    <title-short>JIRD</title-short>
+    <id>http://www.zotero.org/styles/journal-of-international-relations-and-development</id>
+    <link href="http://www.zotero.org/styles/journal-of-international-relations-and-development" rel="self"/>
+    <link href="http://www.zotero.org/styles/journal-of-elections-public-opinion-and-parties" rel="template"/>
+    <link href="http://www.tandf.co.uk/journals/authors/style/reference/ref_fwep.pdf" rel="documentation"/>
+    <author>
+      <name>Michal Mochtak</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issn>1408-6980</issn>
+    <eissn>1581-1980</eissn>
+    <updated>2017-04-22T19:54:57+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" initialize="false"/>
+      <label form="short" prefix=", " suffix=","/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" text-case="capitalize-first" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name and="text" initialize="false" initialize-with="." name-as-sort-order="first"/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text"/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="webpage report" match="any">
+        <text variable="URL" prefix="available at "/>
+        <group prefix=" (" suffix=")">
+          <text term="accessed" text-case="lowercase" suffix=" "/>
+          <date variable="accessed">
+            <date-part name="day" suffix=" "/>
+            <date-part name="month" suffix=", "/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="thesis">
+        <text variable="title"/>
+      </if>
+      <else-if type="bill book graphic legal_case legislation motion_picture song" match="any">
+        <text variable="title" font-style="italic"/>
+      </else-if>
+      <else-if type="report" match="any">
+        <group>
+          <text variable="title" quotes="true"/>
+        </group>
+        <group>
```

Comment:
I would need your help here. Could you do it yourself? As I wrote before, I am new to coding and I ma not sure where exactly I should remove the tags.